datafusion/common/src/hash_utils.rs

```diff
@@ -491,9 +488,6 @@ pub fn create_row_hashes_v2<'a>(
     random_state: &RandomState,
```

Comment:
It looks like we don't use `create_row_hashes` (both versions) anymore?

Reply:
I'm thinking similarly.